sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/EqualPartitionsBalancingStrategy.java

```diff
@@ -53,6 +55,7 @@ public EqualPartitionsBalancingStrategy(String hostName, int minPartitionCount,
 
         int target = this.calculateTargetPartitionCount(partitionCount, workerCount);
         int myCount = workerToPartitionCount.get(this.hostName);
```

Comment:
potential  null pointer when converting integer to int if workerToPartitionCount.get(this.hostName) returns null ?

Reply:
"this.hostname" is guaranteed to be in the map (see calculateTargetPartitionCount())